config/config.go

```diff
@@ -11,10 +11,17 @@ import (
 var ErrInvalidTransport = fmt.Errorf("invalid transport")
 
 type GlobalConfig struct {
-	CachePath string `yaml:"cache_path"`
+	CachePath         string `yaml:"cache_path"`
+	DisableLoopDevice bool   `yaml:"no_loop_device"`
+	BuildkitCacheSize string `yaml:"buildkit_cache_size"`
```

Comment:
I would call this just `cache_size_mb`:
   * the fact that it's buildkit underneath shouldn't matter to the user
   * `_mb` would make the unit of measure clear, so they don't need to go and read the docs. Though I also like being able to say `10G`... I'm torn :-)

What do you think?

Reply:
that's a fair point, because most users don't expect "10GB" to work. I'll go back to just using ints (then it's also one less dependency we have).